src/rm/RequestManagerPoolInfoFilter.cc

```diff
@@ -476,7 +418,8 @@ void RequestManagerPoolInfoFilter::dump(
         int                start_id,
         int                end_id,
         const string&      and_clause,
-        const string&      or_clause)
+        const string&      or_clause,
+        bool               extended)
```

Comment:
Make extended a class attribute so we do not need to pass it around. At the end is a property of the class info method e.g.  VirtualMachinePoolInfoExtended vs  VirtualMachinePoolInfo. The base class can have a bool exteneded attribute set in the class constructor to configure its properties.